README.md

```diff
@@ -331,4 +336,8 @@
 
 ## Project Management
   
-  * [milestones](https://github.com/turbopape/milestones) 
+  * [milestones](https://github.com/turbopape/milestones)
+ 
+## Products
+
+  * [Metabase](http://www.metabase.com/)
```

Comment:
seems, you got me wrong, https://github.com/razum2um/awesome-clojure/blame/master/README.md#L3 such section already exits :)